.rubocop.yml

```diff
@@ -93,8 +93,6 @@ Lint/IdentityComparison: # (new in 0.91)
   Enabled: true
 Lint/MixedRegexpCaptureTypes: # (new in 0.85)
   Enabled: true
-Style/NilLambda: # (new in 1.3)
```

Comment:
Missed this in the big re-alphabetization. Moved below with its `Style` friends.